puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/foreman-packaging-rpm-release.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Does it make sense to include this in https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/foreman-release.yaml since that already controls all the version specific pipelines? It'd mean there's just one place to drop a release and add a new one.

Reply:
agreed.